cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStorageProvider.java

```diff
@@ -56,6 +56,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(PostgreSqlStorageProvider.class);
 
+  private static final String SCAN_FETCH_SIZE_ROWS = "data.storage.sql.scan.size.rows";
```

Comment:
Please move to the io.cdap.cdap.common.conf.Constants alongside other values

Reply:
Done